src/tools/rbd/action/MirrorPool.cc

```diff
@@ -399,6 +503,10 @@ Shell::Action action_info(
   {"mirror", "pool", "info"}, {},
   "Show information about the pool mirroring configuration.", {},
   &get_info_arguments, &execute_info);
+Shell::Action action_status(
+  {"mirror", "pool", "status"}, {},
```

Comment:
On 04/05/2016 10:23 PM, Mykola Golub wrote:
> In src/tools/rbd/action/MirrorPool.cc
> <https://github.com/ceph/ceph/pull/8442#discussion_r58652430>:
>
>> @@ -399,6 +503,10 @@ Shell::Action action_info(
>>    {"mirror", "pool", "info"}, {},
>>    "Show information about the pool mirroring configuration.", {},
>>    &get_info_arguments, &execute_info);
>> +Shell::Action action_status(
>> +  {"mirror", "pool", "status"}, {},
>
> @jdurgin <https://github.com/jdurgin>
>
>     It's probably worth having a general summary command
>
> Yes, thanks for the remainder! I had plans for the summary command. I
> think by default |mirror pool status| will output only summary status,
> and when |--detail| option is specified it will output status for every
> image in the pool

Sounds good

>     Maybe useful to have an option to specify a single image to check on
>     as well
>
> Isn't it what |mirror image status| does?

Ah, yes, somehow missed that